### PR TITLE
Preliminary fix of offset-readout

### PR DESCRIPTION
**Pull request by tamasgal**
This is a preliminary fix for #39. Comments are welcome (my time is very limited though :see_no_evil: )

### CODE REVIEW

File: src/root.jl [hunk 1]
```diff
@@ -223,12 +223,14 @@ end
...
     datas = sizehint!(Vector{UInt8}(), sum(nbytes)) # maximum length if all data are UInt8
-    offsets = sizehint!(Vector{Int32}(), branch.fEntries+1) # this is always Int32
+    offsets = sizehint!(zeros(Int32, 1), branch.fEntries+1) # this is always Int32
+    position = 0
     foreach(branch.fBasketSeek) do seek
         seek==0 && return
         data, offset = readbasketseek(f, branch, seek)
         append!(datas, data)
-        append!(offsets, offset)
+        append!(offsets, offset[2:end] .+ position)
```
Comment (Moelf) [inline src/root.jl:232]:
```suggestion
        append!(offsets, @view(offset, 2:end) .+ position)
```

**Comment by Moelf** [comment]
LGTM

File: src/root.jl (hunk 1 above)
Comment (tamasgal) [reply src/root.jl]: Yep, thanks!

**Comment by tamasgal** [comment]
The `zeros` are there for extra confusion, this  needs a bit of cleanup soon.

One test is failing, I guess assuming that `offset` will have at least 2 elements is too naive `;)`

**Comment by tamasgal** [comment]
I have to figure out how to create a small file with many baskets though, we need a sample file for this.

**Comment by Moelf** [comment]
how about `begin+1:end`

**Comment by tamasgal** [comment]
I am just struggling with the `@view`  and the range containing `begin/end`

**Review by Moelf — APPROVED** [review]

**Pull request merged**